lineman/app/services/inbox_service.coffee

```diff
@@ -0,0 +1,34 @@
+angular.module('loomioApp').factory 'InboxService', (UserAuthService, Records) ->
+  new class InboxService
+    constructor: ->
+      currentUser = UserAuthService.currentUser
+      baseThreadsView = (name) ->
+       view = Records.discussions.collection.addDynamicView(name)
```

Comment:
hey james.. looks like this isnt really ready for review.. thanks for the
feedback but know that it's not at that stage yet

On Wed, 4 Mar 2015 at 20:26 James Kiesel <notifications@github.com> wrote:

> In lineman/app/services/inbox_service.coffee
> <https://github.com/loomio/loomio/pull/1960#discussion_r25756395>:
>
> > @@ -0,0 +1,34 @@
> > +angular.module('loomioApp').factory 'InboxService', (UserAuthService, Records) ->
> > +  new class InboxService
> > +    constructor: ->
> > +      currentUser = UserAuthService.currentUser
> > +      baseThreadsView = (name) ->
> > +       view = Records.discussions.collection.addDynamicView(name)
>
> Should clean this up before merge
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1960/files#r25756395>.
>